docs/api/cluster/elbow.rst

```diff
@@ -31,21 +33,25 @@ However, two other metrics can also be used with the ``KElbowVisualizer`` -- ``s
 The ``KElbowVisualizer`` also displays the amount of time to train the clustering model per :math:`K` as a dashed green line, but is can be hidden by setting ``timings=False``. In the following example, we'll use the ``calinski_harabaz`` score and hide the time to fit the model.
 
 .. plot::
+    :context: close-figs
+    :alt: KElbowVisualizer on synthetic dataset with 8 random clusters
 
     from sklearn.cluster import KMeans
     from sklearn.datasets import make_blobs
 
     from yellowbrick.cluster import KElbowVisualizer
 
-    # Create synthetic dataset with 8 random clusters
-    X, _ = make_blobs(centers=8, n_features=12, shuffle=True, random_state=42)
+    # Generate synthetic dataset with 8 random clusters
+    X, y = make_blobs(n_samples=1000, n_features=12, centers=8, random_state=42)
```

Comment:
I replaced `_` with `y` here, since even though the target variable is not needed and it doesn't affect the final plot, I thought `_` could be confusing for new users who may not have seen this before. In addition, it's not used in the other examples in our documentation when `make_blobs` is used to create the dataset.

I also removed `shuffle=True` and added `n_samples=1000` for the same reasons I described above in a previous comment.